packages/block-editor/src/components/index.js

```diff
@@ -16,7 +16,12 @@ export { default as BlockEdit, useBlockEditContext } from './block-edit';
 export { default as BlockFormatControls } from './block-format-controls';
 export { default as BlockIcon } from './block-icon';
 export { default as BlockNavigationDropdown } from './block-navigation/dropdown';
-export { default as __experimentalBlockNavigationList } from './block-navigation/list';
+export {
+	default as __experimentalBlockNavigationList,
+	BlockNavigationContext as __experimentalBlockNavigationContext,
```

Comment:
I think this no longer needs to be exported now that it's only used internally in the BlockNavigation components